### PR TITLE
Mayhem

### PR DESCRIPTION
**Pull request by ennamarie19**

### CODE REVIEW

File: Mayhemfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+image: ghcr.io/ennamarie19/gphoto2-fuzz:latest
```
Comment (xansec) [inline Mayhemfile]: Please omit image name.

**Review by xansec — CHANGES_REQUESTED** [review]
Can you explain what all of the changes under the `./po` directory are for? They are pretty large diffs so I can't look into them. Maybe a mistaken commit?

**Comment by ennamarie19** [comment]
@xansec Done!

**Comment by xansec** [comment]
LGTM!

**Pull request merged**